stable/prometheus/templates/server-deployment.yaml

```diff
@@ -26,6 +26,10 @@ spec:
         release: {{ .Release.Name }}
     spec:
       serviceAccountName: {{ if .Values.rbac.create }}{{ template "prometheus.server.fullname" . }}{{ else }}"{{ .Values.server.serviceAccountName }}"{{ end }}
+    {{- if .Values.alertmanager.imagePullSecrets }}
```

Comment:
I believe this should be `.Values.server.imagePullSecrets`. However I think the you should move `alertmanager.imagePullSecrets` to `alertmanager.image.pullSecrets` and `server.imagePullSecrets` to `server.image.pullSecrets`